services/discourse/discourse.service.js

```diff
@@ -21,20 +25,19 @@ class DiscourseBase extends BaseJsonService {
   static buildRoute(metric) {
     return {
       base: 'discourse',
-      // Do not base new services on this route pattern.
-      // See https://github.com/badges/shields/issues/3714
-      pattern: `:scheme(http|https)/:host/${metric}`,
+      pattern: `${metric}`,
```

Comment:
```suggestion
      pattern: metric,
```

Reply:
Good spot! That's what I get for being lazy with the delete key 😉